scripts/set_velocity_aberration.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python

```

Comment:
Hmm, `scripts/*` are installed to `$venv/bin` as executable scripts so wouldn't removing this be bad?

Reply:
Doh! Yes.  Since these are called directly.

Because `jplephem` wasn't imported properly this was showing up as different from the other scripts that were getting that line rewritten upon installation.  I'll will change this back.